iron-ajax.html

```diff
@@ -452,6 +457,20 @@
         this._discardRequest.bind(this, request)
       );
 
+      var event = new CustomEvent('iron-ajax-presend', {
```

Comment:
We still have not decided what we are going to do in regards to firing events in pure 2.0 elements. Can you please stick with `this.fire` for now? It should default to `bubbles: true` and `composed: true`.